### DISCUSSION TITLE
What is the best way to discuss issues in nightly builds

### DISCUSSION THREAD

**Discussion by ar4development** [carla-simulator/carla#4176] [Q&A]
Hi,

What if one observes the issue or strange behavior in nightly build? What is the best (aka effective) way to highlight and discuss them? On one hand they might be fixed in later dev builds, on other they might leak to release. Issues section on github is for releases, so what would be the proper way to discuss dev builds?

Thanks!

**Comment by corkyw10** [comment]
Hi @ar4development, you can use GitHub issues for that as well. Just specify that you are using the nightly build.
